src/window_manager.cc

```diff
@@ -809,7 +810,9 @@ void _GNW_win_refresh(Window* window, Rect* rect, unsigned char* dest)
     }
 
     if ((window->flags & WINDOW_TRANSPARENT) && _buffering && !_doing_refresh_all) {
-        // TODO: Incomplete.
+        Rect dirtyRect = window->rect;
+        windowRefreshAll(&dirtyRect);
+        return;
```

Comment:
instead of this, I think we can just remove this `if` (and so always do the block below).